h2/stream.py

```diff
@@ -874,6 +901,9 @@ def advertise_alternative_service(self, field_value):
         Advertise an RFC 7838 alternative service. The semantics of this are
         better documented in the ``H2Connection`` class.
         """
+        self.config.logger.debug(
+            "Advertise alternative service of %s for %r", field_value, self
+        )
```

Comment:
`field_value` is a `bytes` object, which means you probably want to print the repr for consistent behaviour on all platforms.